src/buttons/CopyToClipboardButton/Readme.md

```diff
@@ -0,0 +1,29 @@
+# CopyToClipboardButton
+
+Button that copy the given value to the clipboard and provide 3 steps feedback for user
+
+- Copy (default status)
+- Copying... (work in progress)
```

Comment:
(nit) please change `(work in progress)` to: `(Working/In progress)` which is more clear